creativecloud/deps/merch-sidenav.js

```diff
@@ -1,4 +1,3 @@
-// branch: main commit: d113c9c27640ef88ab28eff3ef4637919479e540 Wed, 03 Jul 2024 15:04:49 GMT
```

Comment:
@adgaurgit please update this dependency with latest main build (with that header).
you'll find it from running `npm run build:milo` and picking dep from milo-libs folder in mas project